.github/workflows/pytest.yml

```
@@ -3,6 +3,11 @@ name: Pytest
 on:
   pull_request:
     types: [opened, synchronize, reopened]
+    paths:
+      - "**.py"
+      - Dockerfile
+      - requirements_dev.txt
+      - "saleor/**"
```

```suggestion
      - "saleor/**"
      - ".github/workflows/pytest.yml"
```

..in case you change the way you run the tests.